sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/init.sql

```diff
@@ -0,0 +1,115 @@
+CREATE TEMP FUNCTION sum_map_values(map ARRAY<STRUCT<key STRING, value INT64>>)
+RETURNS INT64 AS (
+  (SELECT SUM(value) FROM UNNEST(map))
+);
+
+CREATE OR REPLACE TABLE
+  fenix_derived.attributable_clients_v1
+PARTITION BY
+  submission_date
+CLUSTER BY
+  adjust_network,
+  adjust_campaign,
+  sample_id AS (
+    WITH client_day AS (
+      SELECT
+        DATE(submission_timestamp) AS submission_date,
+        sample_id,
+        client_info.client_id,
+        LOGICAL_OR(
+          mozfun.norm.extract_version(client_info.app_display_version, 'major') >= 107
+        ) AS has_search_data,
+        SUM(sum_map_values(metrics.labeled_counter.browser_search_with_ads)) AS searches_with_ads,
+        SUM(sum_map_values(metrics.labeled_counter.browser_search_ad_clicks)) AS ad_clicks,
```

Comment:
This bit is to pull in the data from baseline ping for search only when the client is above 107 on that day.